setup.py

```diff
@@ -63,10 +63,12 @@ def get_requirements(path):
     tests_require=get_requirements("dev-requirements.txt"),
     extras_require={"sqla": ["sqlalchemy", "sqlalchemy-utils"], "signals": ["blinker"]},
     cmdclass={"test": PyTest},
+    entry_points={"pytest11": ["pytest_flask_dance = flask_dance.fixtures.pytest"]},
```

Comment:
This magical line allows others to use this fixture in their tests. [See the pytest docs on writing plugins.](https://docs.pytest.org/en/latest/writing_plugins.html#making-your-plugin-installable-by-others)

Reply:
I was very confused about that line until I found the docs, the naming is fairly unobvious.